src/parser/scanner_support.c

```diff
@@ -59,6 +60,13 @@ extern int yycolumn_end;
 
 
 /* matching hint information to output*/
+typedef struct st_stmt_hint STMT_HINT;
```

Comment:
Isn't there a convention that struct type names are capitalized for struct names?

Reply:
The company code convention rules are not clear.
However, in the previous example, it was used as follows.

```
typedef struct parser_node PT_NODE;
typedef struct pt_alter_info PT_ALTER_INFO;
```

Reply:
Yes, I understand.